ci-operator/step-registry/ipi/conf/ibmcloud/custom-endpoints/ipi-conf-ibmcloud-custom-endpoints-ref.yaml

```diff
@@ -43,6 +43,26 @@ ref:
     default: ""
     documentation: |-
       GlobalTagging endpoint
+  - name: SERVICE_ENDPOINT_COSConfig
+    default: ""
+    documentation: |-
+      COSConfig endpoint
+  - name: SERVICE_ENDPOINT_GlobalCatalog
+    default: ""
+    documentation: |-
+      GlobalCatalog endpoint
+  - name: SERVICE_ENDPOINT_KeyProtect
+    default: ""
+    documentation: |-
+      KeyProtect endpoint      
+  - name: SERVICE_ENDPOINT_CIS
+    default: ""
+    documentation: |-
+      CIS endpoint, which is used in External cluster.
+  - name: SERVICE_ENDPOINT_HyperProtect
```

Comment:
I did not see the default endpoint for HyperProtect defined in eps_default.json, if we are not ready to test it, pls remove it.